lib/AppInfo/Application.php

```diff
@@ -156,11 +156,14 @@ public function updateCSP() {
 	}
 
 	public function checkAndEnableCODEServer() {
-		// Supported only on Linux OS, and x86_64 platform
-		if (PHP_OS_FAMILY !== 'Linux' || php_uname('m') !== 'x86_64')
+		// Supported only on Linux OS, and x86_64 & ARM64 platforms
+		$supportedArchs = array('x86_64', 'aarch64');
+		if (PHP_OS_FAMILY !== 'Linux' || !in_array(php_uname('m'), $supportedArchs))
```

Comment:
```suggestion
		$osFamily = PHP_VERSION_ID >= 70200 ? PHP_OS_FAMILY : PHP_OS;
		if (osFamily !== 'Linux' || !in_array(php_uname('m'), $supportedArchs))
```

Not an issue of this PR but would be good to have it like this, otherwise there will be an error on PHP 7.1.